actor/echo-messaging/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+use wasmbus_rpc::actor::prelude::*;
+use wasmcloud_interface_messaging::*;
+
+#[derive(Debug, Default, Actor, HealthResponder)]
+#[services(Actor, MessageSubscriber)]
+struct EchoMessagingActor {}
+
+#[async_trait]
+impl MessageSubscriber for EchoMessagingActor {
```

Comment:
This struct and impl should be `{{to_pascal_case project-name}}Actor` so it takes the name that the user supplies